src/api.js

```diff
@@ -0,0 +1,14 @@
+import axios from 'axios'
+
+const newsApi = axios.create({
+  baseURL: "https://news-13r8.onrender.com/api"
+});
+
```

Comment:
Great use of axios baseURL functionality to keep code tide, and separation into an `api.js` file ✅